tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/org_mozilla_ios_focus.metrics.schema.json

```diff
@@ -0,0 +1,191 @@
+[
+    {
+      "fields": [
+        {
+          "type": "STRING",
+          "name": "android_sdk_version"
```

Comment:
Probably leftover from copy-paste

Reply:
This column is present in the ios focus/klar tables. I assumed this has been done for standardization purposes.

Reply:
Yes, I can confirm that sadly that's the case :)

Reply:
Gotcha 😅